ts/Extensions/DashboardPlugins/HighchartsComponent.ts

```diff
@@ -698,7 +698,11 @@ class HighchartsComponent extends Component {
         if (this.chartConstructor !== 'chart') {
             const factory = charter[this.chartConstructor] || G.chart;
             if (factory) {
-                return factory(this.chartContainer, this.chartOptions);
+                try {
+                    return factory(this.chartContainer, this.chartOptions);
+                } catch {
+                    // returns errors from factory by default
```

Comment:
Silent catch is bad design. At least log the error.

Reply:
Fixed.